lib/controllers/videoListController.dart

```diff
@@ -35,8 +35,13 @@ class VideoListController extends GetxController {
   Map<String, Image> imageCache = {};
   ScrollController scrollController = ScrollController();
   VideoListErrors error = VideoListErrors.none;
+  final bool shouldRefetch;
 
-  VideoListController(this.videoList);
+  VideoListController({required this.videoList, this.shouldRefetch = false}) {
+    if (shouldRefetch && videoList.hasRefresh()) {
```

Comment:
This should be in the  onReady() method. if done here and the refresh is completed before the  ui is properly loaded it will crash. onReady is the same as onInitState in getx